x-pack/test/functional/apps/dashboard/group3/drilldowns/index.ts

```diff
@@ -16,12 +16,15 @@ export default function ({ loadTestFile, getService }: FtrProviderContext) {
 
     before(async () => {
       await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional');
```

Comment:
if this archive is not used anywhere else (I don't know if that's true or not) I would include it in this PR since it makes more sense this way.

Reply:
@marius-dr that's data archive. Not  .kibana. So no changes necessary. Thanks!